dataframely/collection.py

```diff
@@ -903,6 +925,24 @@ def _validate_input_keys(cls, data: Mapping[str, FrameType], /) -> None:
             )
 
 
+def read_parquet_metadata_collection(
```

Comment:
Are you moving the `read_parquet_metadata` method we discussed to another PR? :smile: